installer/packer/scripts/kov/configure_kov.sh

```diff
@@ -0,0 +1,179 @@
+#!/usr/bin/bash
```

Comment:
wait. I think user can change those variables after restart. so we still have to do the same thing here(using a sed command to replace old value or through other tools)..